src/server.rs

```diff
@@ -44,9 +51,41 @@ pub fn make_server_endpoint(bind_addr: SocketAddr) -> Result<(Endpoint, Vec<u8>)
     Ok((endpoint, server_cert))
 }
 
+#[derive(Deserialize, Debug)]
+struct ServerConf {
+    proxy: HashMap<String, SocketAddr>,
+}
+impl ServerConf {
+    fn new() -> Self {
+        ServerConf {
+            proxy: HashMap::<String, SocketAddr>::new(),
+        }
+    }
+}
+
 #[tokio::main]
 pub async fn run(options: Opt) -> Result<(), Box<dyn Error>> {
+    let conf: ServerConf = match options.conf_path {
+        Some(path) => {
+            info!("[server] importing conf file: {}", path.display());
+            toml::from_str(&(read_to_string(path).await?))?
+        }
+        None => ServerConf::new(),
+    };
+
+    let default_proxy = match conf.proxy.get("default") {
+        Some(sock) => sock.clone(),
+        None => {
+            use std::net::IpAddr::V4;
```

Comment:
place this line to the head of this file?

Reply:
`SocketAddr` also has `V4`, and maybe they will collide in namespace, when to use both of them. how about importing `std::net::IpAddr` in head and use `V4` as `IpAddr::V4` ?

Reply:
SGTM

Reply:
I found more concise way
https://github.com/JyJyJcr/quicssh-rs/blob/07b832910cbcb87e1d6c30ac509ea035448a0386/src/server.rs#L80